src/renderer/features/network/NetworksList/ui/NetworksList.tsx

```diff
@@ -1,44 +1,23 @@
-import { useEffect, useState, ReactNode } from 'react';
+import { ReactNode } from 'react';
 
-import { ExtendedChain, networkUtils } from '@entities/network';
+import { ExtendedChain } from '@entities/network';
 import { CaptionText, Counter, Accordion } from '@shared/ui';
+import { networksListUtils } from '../lib/networks-list-utils';
 
 type Props = {
   title: string;
   isDefaultOpen?: boolean;
-  query?: string;
   networkList: ExtendedChain[];
   children: (network: ExtendedChain) => ReactNode;
 };
 
-export const NetworkList = ({ title, isDefaultOpen, query, networkList, children }: Props) => {
-  const [isListOpen, setIsListOpen] = useState(isDefaultOpen);
-
-  useEffect(() => {
-    if (query) {
-      setIsListOpen(Boolean(query));
-    } else {
-      setIsListOpen(isDefaultOpen);
-    }
-  }, [query]);
-
+export const NetworksList = ({ title, isDefaultOpen, networkList, children }: Props) => {
```

Comment:
I think we should have a convention for naming. Because I prefer `NetworkList` for this

Reply:
agreed, I'll change